client-core/src/wallet/syncer.rs

```diff
@@ -449,8 +450,9 @@ impl<
         }
 
         let (target_height, target_app_hash, target_block_hash) =
-            if self.env.options.enable_fast_forward {
+            if self.env.options.enable_fast_forward || self.env.options.disable_light_client {
                 (
+                    /* use api api directly */
```

Comment:
`use api directly` ?

Reply:
yes, not using light client protocol

Reply:
the comment says "use **api api** directly" -- maybe "use Tendermint RPC directly" is clearer ?

Reply:
that's better,i'll change